src/test/scala/org/broadinstitute/dsde/agora/server/webservice/AgoraImportSpec.scala

```diff
@@ -67,24 +86,28 @@ class AgoraImportSpec extends ApiServiceSpec with FlatSpecLike{
     }
   }
 
-  "MethodsService" should "return a 400 when copying a method and specifying a WDL with an invalid import statement" in {
+  "MethodsService" should "return a 500 when copying a method and specifying a WDL with an invalid import statement" in {
     Post(copyUrl, copyPayload(testBadAgoraEntityInvalidWdlImportFormat.payload)) ~>
       methodsService.querySingleRoute ~> check {
-      assert(status == BadRequest)
+      assert(status == InternalServerError)
+      assert(body.asString contains "Failed to import workflow invalid_syntax_for_tool.:")
     }
   }
 
-  "MethodsService" should "return a 400 when copying a method and specifying a WDL with an import statement that references a non-existent method" in {
+  "MethodsService" should "return a 500 when copying a method and specifying a WDL with an import statement that references a non-existent method" in {
     Post(copyUrl, copyPayload(testBadAgoraEntityNonExistentWdlImportFormat.payload)) ~>
       methodsService.querySingleRoute ~> check {
-      assert(status == BadRequest)
+      assert(status == InternalServerError)
+      assert(body.asString contains "Failed to import workflow broad.non_existent_grep.1.:")
     }
   }
 
-  "MethodsService" should "return 400 when copying a method and specifying a WDL that contains an import, because imports are no longer allowed" in {
+  "MethodsService" should "return 200 when copying a method and specifying a valid WDL that contains an import" in {
     Post(copyUrl, copyPayload(testEntityWorkflowWithExistentWdlImport.payload)) ~>
       methodsService.querySingleRoute ~> check {
-      assert(status == BadRequest)
+      assert(status == OK)
+      assert(body.asString contains "\"name\": \"testMethod1\"")
+      assert(body.asString contains "\"createDate\"")
```

Comment:
How about one more unit test that checks the use case of a WDL with syntactically valid http import(s), but the urls in those imports return 404s? This seems like a case users will run into not infrequently, and we might start adding code changes around it; it would be good to have a test case for it from the beginning.